### PR TITLE
Create api.jamied132.json

### PR DESCRIPTION
**Pull request by JamieD132**
This is my pull request to get an NS record. The reason are multiple reasons why I would like this:
 - I want to make a website where the main site is at jamied132.is-a.dev, and the api part of the site is api.jamied132.is-a.dev (because it looks more professional than api.jamied132.workers.dev)
 - At my school the workers.dev domain is blocked so i am incapable of editing/testing or even visiting the api or my actual site (because it relies on the api) while i am at school
You may have noticed that i left the ns record blank (i *think* that that is possible?) the reason is because i do not know what nameservers to add to those yet until the actual domain is created an cloudflare will tell me. if this is not possible then i can temporarily give it another record or something like that and then change it. 

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/api.jamied132.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Subdomain for the api of my website",
+  "owner": {
+    "username": "JamieD132",
+    "email": "jamied132.email@gmail.com",
+    "darflen.com": "JamieD132"
+  },
+  "record": {
+    "NS": ["", ""]
```
Comment (CuteDog5695) [inline domains/api.jamied132.json:9]: You know you can try and add the domain to Cloudflare and from there to get the NS records, right?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/api.jamied132.json (hunk 1 above)
Comment (JamieD132) [reply domains/api.jamied132.json:9]: uhhh no i probably don't know that, i am quite new to this :/ what do you mean? Do you mean add jamied132.is-a.dev to cloudflare and from once ive done that give it a api. subdomain?

Comment (CuteDog5695) [reply domains/api.jamied132.json:9]: Add the main subdomain to Cloudflare, get the NS records and add the NS records to your **main domain**, not your API subdomain.

Comment (JamieD132) [reply domains/api.jamied132.json:9]: ok...

Comment (JamieD132) [reply domains/api.jamied132.json:9]: did i do it right? https://github.com/is-a-dev/register/pull/16899

**Comment by CuteDog5695** [comment]
I'll close this PR since its a duplicate

**Pull request closed without merging**